randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -1301,21 +1332,24 @@ Extra - room_id: [46]
   * Extra - door_idx: (114,)
   > Door to Flooded Tower
       Any of the following:
-          Charge Beam or Missiles ≥ 10 or Combat (Intermediate) or Normal Damage ≥ 190 or Can Use Power Bombs
+          Charge Beam or Missiles ≥ 10 or Combat (Intermediate) or Can Use Power Bombs
```

Comment:
charge should be changed to charge + beam upgrade, missing grav+screw

Reply:
not missing gravscrew

Reply:
Can you not kill the pirates/fishies with screw attack? SA is at least part of the common template for killing tough enemies.

Reply:
no, I meant grav and screw is listed two lines under this comment

Reply:
👨‍🦯